flocker/node/_docker.py

```diff
@@ -208,13 +208,14 @@ def exists(unit_name):
             otherwise ``False``.
         """
 
-    def remove(unit_name):
+    def remove(unit_name, force):
```

Comment:
I'd have thought `zope.verify` would have caught this, but the signature in the interface should match the signature in the implementation, so the interface ought to have `force=False`.